esmvalcore/cmor/_fixes/cmip6/gfdl_cm4.py

```diff
@@ -0,0 +1,66 @@
+"""Fixes for CESM2 model."""
```

Comment:
One last doc string and we are good to go.

Reply:
Right... ;)